### PR TITLE
[nbc:NBCSportsVPlayer] Corrected video URL (fixes issue #13873)

### PR DESCRIPTION
**Pull request by parmjitv**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

Updated the extractor to use the correct video URL and replaced the broken tests - issue #13873.

Cheers and thank you!

Parmjit V.

### CODE REVIEW

**Comment by parmjitv** [comment]
@dstftw - Does this PR require any corrections?  I believe the fix for this issue is pretty straightforward.  Thanks!

**Comment by parmjitv** [comment]
@dstftw - Are any collaborators available for a first review of this PR?  Much appreciated!

File: youtube_dl/extractor/nbc.py [hunk 1]
```diff
@@ -109,18 +109,18 @@ class NBCSportsVPlayerIE(InfoExtractor):
     _VALID_URL = r'https?://vplayer\.nbcsports\.com/(?:[^/]+/)+(?P<id>[0-9a-zA-Z_]+)'
 
     _TESTS = [{
-        'url': 'https://vplayer.nbcsports.com/p/BxmELC/nbcsports_share/select/9CsDKds0kvHI',
+        'url': 'http://vplayer.nbcsports.com/p/BxmELC/chicago_embed/select/media/u4xMi3ngFKsS',
```
Comment (dstftw) [inline youtube_dl/extractor/nbc.py]: https://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/9CsDKds0kvHI is still available.

File: youtube_dl/extractor/nbc.py [hunk 2]
```diff
@@ -109,18 +109,18 @@ class NBCSportsVPlayerIE(InfoExtractor):
...
+            'ext': 'mp4',
+            'description': 'md5:8fb3edf07c77cc5901c5f37fa019f87c',
+            'title': 'Cubs PGL: Who needs to step up in Contreras\' absence',
+            'timestamp': 1502319595,
+            'upload_date': '20170809',
             'uploader': 'NBCU-SPORTS',
         }
     }, {
-        'url': 'http://vplayer.nbcsports.com/p/BxmELC/nbc_embedshare/select/_hqLjQ95yx8Z',
+        'url': 'https://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/media/qyc_v6mfDyWv',
```
Comment (dstftw) [inline youtube_dl/extractor/nbc.py]: As well as this.

Comment (parmjitv) [reply youtube_dl/extractor/nbc.py]: https://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/_hqLjQ95yx8Z

ERROR: This content expired on 9/17/14 12:23 PM.

**Comment by parmjitv** [comment]
@dstftw - The test suite has been updated as requested.  Please let me know if there are any further corrections that need to be made.  Cheers!

File: youtube_dl/extractor/nbc.py (hunk 2 above)
Comment (dstftw) [reply youtube_dl/extractor/nbc.py]: http://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/_hqLjQ95yx8Z

Comment (dstftw) [reply youtube_dl/extractor/nbc.py]: It's a matching only test. Testing two identical URLs twice does not make any sense.

Comment (parmjitv) [reply youtube_dl/extractor/nbc.py]: Gotcha.  I will remove the duplicate test.

**Comment by parmjitv** [comment]
@dstftw - The duplicate test has been removed, please review at your convenience.  Thank you!

File: youtube_dl/extractor/nbc.py (hunk 2 above)
Comment (dstftw) [reply youtube_dl/extractor/nbc.py]: You don't get it. **Don't touch tests**. After you change there are two tests with identical scheme. Testing this does not make any sense. Each tests must test **something special**.

Comment (parmjitv) [reply youtube_dl/extractor/nbc.py]: My apologies, a misunderstanding on my part.  With my latest commit, there is now only one test in place for this extractor - shall I revert the test suite back to the original state?  I believe those URLs were broken to begin with.

Comment (dstftw) [reply youtube_dl/extractor/nbc.py]: Change 1 test to https://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/9CsDKds0kvHI.
Change 2 test to https://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/media/_hqLjQ95yx8Z.

Comment (parmjitv) [reply youtube_dl/extractor/nbc.py]: Ah, now I see what you mean!  Thanks for the clarification.

**Comment by parmjitv** [comment]
@dstftw - The test suite has been updated with the correct URLs, thanks for your patience.  Please let me know if there is anything else still outstanding for this PR.

**Pull request merged**